README.md

```diff
@@ -154,6 +154,7 @@ You can pass options either via command line (in [kebab-case](https://lodash.com
 
 These options are currently available:
 - `branch`: The branch on which releases should happen. Default: `'master'`
+- `no-pre-release`: By default, the releases with a npm dist-tag other than `latest` will produce a [Github release](https://help.github.com/articles/creating-releases/) with the __pre-release__ tag. You can add any dist-tag in the `no-pre-release` array in order to produce a regular release (non pre-release) on Github for those dist-tag. Default: `['latest']`
```

Comment:
@pvdlg @Apidcloud are you sure we need that flag? What are current use cases for it? I'm reluctant to add more configuration if we can avoid it

Reply:
The use case is to force a regular release on Github, even if the release is done on a dist-tag other than `latest`.
Basically when you release on `v2` but you still want the release on Github to not be tagged `pre-release`.

Might be useful when you release on a legacy/LTS/old version dist-tag.

Reply:
> Might be useful when you release on a legacy/LTS/old version dist-tag.

I think this is a separate use case that we should address separately. My understanding is that this PR is all about setting the pre-release flag on releases when publishing to a custom tag?

We don't really support LTS releases yet, do we? That is a valid use case by itself but I think it will need other features like a built-in restriction of not allowing breaking feature releases and using a major version as base, so that even if v3.0.0 is already released, it can still release v2.0.1

Reply:
> We don't really support LTS releases yet, do we?

Kind of. You can have a branch configured to release on the dist-tag `lts` for example. In that case you might not want the prerelease tag.
But yes in order to support it _properly_  we would need a to implement a verification to make sure we don't release something out of bound.

So I guess when we do that, we would have everything necessary to figure out it a LTS branch therefore the `prerelease` tag should be `false`.

In that case yes it makes the `no-pre-release` param useless.

Kind of feel bad that I asked @Apidcloud to add it...

Reply:
> Kind of feel bad that I asked @Apidcloud to add it...

I know, I understand, but remember Open Source is free as in puppy, once we take it, we will have to maintain it until the end of time :) Better have a longer, uncomfortable discussion upfront, that's how it goes some times, time gets spent on work only to realize it won't get merged. Still better than the alternative of accepting changes and than maintaining them for forever ;)

Reply:
also just to make that clear, I want to make LTS releases a first-class feature of semantic-release, so we can leave it safely out of scope for this feature